cirq-web/cirq_ts/README.md

```diff
@@ -0,0 +1,91 @@
+## Cirq Typescript Development
+
+*This directory contains the code and instructions for visualization tools in a web browser or Colab/Juptyer notebooks. We do this using Typescript. While necessary, this is only half of the code needed to run the `cirq-web` project. For information on how to integrate projects here with Python and the wider Cirq package, see the `cirq_web` directory.*
+
+Visualizations run on [NodeJS](https://nodejs.org/en/), and we use [npm](https://www.npmjs.com/) for package management. To start developing, clone the repository and run `npm install` within this directory to install the necessary packages and begin development. You will need to install Node and npm if you haven't already.
```

Comment:
Is there any way to run the `npm` instructions from the top-level Cirq directory instead? Not mandatory, but it would be convenient.

Also may be useful to document known error messages in `npm install`. I got this:
```
4 packages are looking for funding
  run `npm fund` for details
3 moderate severity vulnerabilities
To address all issues (including breaking changes), run:
  npm audit fix --force
Run `npm audit` for details.
```
It's unclear if those messages should inspire worry.

Reply:
I think me and Balint discussed moving to the top at one point..we're opting keep NodeJS and npm stuff within `cirq_ts` since we want cirq-web in general to be an opt in feature of Cirq. I'm pretty sure if we had it in the top-level directory than you would need to have it as a dependency..correct me if there's a way around it. 

There actually is a way that you can run `npm` instructions from the top level though, if you ran `npm --prefix 'cirq-web/cirq_ts'` you should be able to interact with everything as if you were in the `cirq_ts` directory. I can add this into the documentation

Reply:
Including the `npm --prefix 'cirq-web/cirq_ts'` note would be nice as a convenience item.

Reply:
Done

Reply:
This still reads as `npm install`. I would suggest implementing `check/npm` with the prefix included.